vipps-order-management-api.md

```diff
@@ -180,72 +78,208 @@ everyone sends it too. It can speed up any trouble-shooting quite a bit.
+    - `POST:/ecomm/v2/payments` 
+2. Save the **orderId** you used when initiating
+3. Add an image (optional)
+    - `POST:/v1/images` 
+4. Save the imageId
+5. Add a category with link and image
+    - `/v2/{paymentType}/categories/{orderId}`
+6. Add a receipt 
+    - `/v2/{paymentType}/receipts/{orderId}`
 
-## Assisted
 
-Vipps offers assisted content monitoring as a way for Merchants to deal with the regulatory demands of content monitoring. For some merchants Vipps can utilize the merchant's webpage for content monitoring, continuously verifying that the actual products being sold coincides with the expected products.
+# Categories
+* [API Documentation with examples here](https://vippsas.github.io/vipps-order-management-api/redoc.html#tag/Category)
 
-### Mandatory use case
+The following section will explain how to enrich a Vipps transaction with `Categories` and `Images`. A `Link` and `Category` is required when using this API,
+whereas `Images` are optional.
+
+## Category
+In order to provide customers with more up to date information about their order,
+you can add a `Category` with a URL to the Vipps Transaction that can take the
+customer to a location on your website. Links are activated when a customer
+clicks the link in the transaction page in the Vipps App. The mobile device's standard web browser will open and the user is redirected to the link location. Below you can se an example of a Vipps transaction with the "Shipping information" `Category`.
 
-However if you as a merchant does not have a permanent website that can be utilized for content monitoring, for example you do not have a user facing website or the website is ephemeral/short lived then you must utilize Vipps Assisted Content Monitoring.
+<p align="center">
+  <img src="images/order-link-shipping-information.png" alt="Shipping information link" width="250">
+</p>
 
-### Guidance
+The category will determine how the app handles the link, additional information and push. We currently support these categories:
+| Category                      | Description                                                        |
+| ----------------------------- | ------------------------------------------------------------------ |
+| `Receipt`                     | A link to a location where the customer can access and download a valid proof of purchase and receipt for this particular order     |
+| `Order Confirmation`          | A link to a location that contains information and status of the order. If your webshop or site has a "My orders" page or similar - this link category can take the customer there |
+| `Delivery information`        | A link to a location that contains information and status about the shipping or delivery related to the order. This could be a link to hosted by your freight carrier, or a link to your site. If your webshop or site has a "My order" page that includes delivery related information about the order - this link category can be used |
```

Comment:
"related information about the order - this link category "  -> use comma instead.  Same for the previous line.
I would have put periods at the end of each of these lines......